docs/content/users/quickstart.md

```diff
@@ -903,7 +903,7 @@ If you have any questions there is lots of help in the [DDEV thread in the Proce
     mkdir my-shopware-site && cd my-shopware-site
     ddev config --project-type=shopware6 --docroot=public
     ddev start
-    ddev composer create-project "shopware/production:^v6.5"
+    ddev composer create-project "shopware/production:6.5"
```

Comment:
We should be using the upstream recommendation, and skipping the version on there. 

```suggestion
    ddev composer create-project "shopware/production"
```